CONTRIBUTING.md

```diff
@@ -189,124 +93,106 @@ require.Equal(t, expectedTx[:32], calculatedTx[:32],
 "First 32 bytes of the txs differed. tc #%d, i #%d", tcIndex, i)
 ```
 
-## Branching Model and Release
+### Pull Requests
 
-User-facing repos should adhere to the [trunk based development branching model](https://trunkbaseddevelopment.com/).
+Before submitting a pull request:
 
-Libraries need not follow the model strictly, but would be wise to.
+* merge the latest main `git merge origin/main`,
+* run `make lint run-tests` to ensure that all checks and tests pass.
```

Comment:
The target `run-tests` is effectively unit tests. 
Nomenclature should change to `run-unit-tests` to make it obvious to contributors not familiar with the codebase.

The target `test-e2e` are integration tests, these are run on Github. Substantiate work should include integration tests. We need to have guidelines for this.

Reply:
Yes, we should update the Makefile. At the moment is very unclear.